### ISSUE TITLE
GitHub Releases No Longer Publishing

### ISSUE THREAD

**Issue by tubbo** [twurple/twurple#381]
## Bug Report

We're not seeing changelog diffs when we update Twurple with Renovate:

![Screen Shot 2022-08-03 at 12 49 23 PM](https://user-images.githubusercontent.com/113026/182664826-d105a0f3-2ab7-46d4-8364-ce91d83632c1.png)

It appears the last release on GitHub was v5.1.5, so I'm thinking maybe the changelog generation broke? This is a lot easier to read than combing through 100+ commit messages, especially for a minor release. Lerna should be doing this for you automatically when you publish, but you have to set a `$GITHUB_TOKEN` in the env so it can communicate with GitHub's API. Happy to help if needed, we use these libraries every day and they are really nice!

**Comment by d-fischer** [comment]
I never do generation. I do these manually. Will follow up later today.

**Comment by tubbo** [comment]
@d-fischer No worries, I read through the publish script and noticed you were using Lerna, so I figured something got out of sync. Anyway, if you are doing these releases manually, just be aware that automatic dependency update tools may be sending out PRs that look like this, and causing developers to have to hunt through 100+ commits and/or happen to remember that the GItHub release was added later. Not a huge deal for patches, but for minor releases it's a bit more of a burden.
